cmd/kubeadm/app/phases/markcontrolplane/markcontrolplane.go

```diff
@@ -28,7 +28,8 @@ import (
 // MarkControlPlane taints the control-plane and sets the control-plane label
 func MarkControlPlane(client clientset.Interface, controlPlaneName string, taints []v1.Taint) error {
 
-	fmt.Printf("[mark-control-plane] Marking the node %s as control-plane by adding the label \"%s=''\"\n", controlPlaneName, constants.LabelNodeRoleMaster)
+	fmt.Printf("[mark-control-plane] Marking the node %s as control-plane by adding the labels \"%s=''\" and \"%s=''\"\n",
+		controlPlaneName, constants.LabelNodeRoleMaster, constants.LabelNodeRoleControlPlane)
```

Comment:
```suggestion
	fmt.Printf("[mark-control-plane] Marking the node %s as control-plane by adding the labels \"%s=''\" and \"%s='' (deprecated)\"\n",
		controlPlaneName, constants.LabelNodeRoleControlPlane, constants.LabelNodeRoleMaster)
```
Let's make it explicit for the users that the old label is now deprecated